server/src/main/java/org/opensearch/common/cache/stats/ImmutableCacheStats.java

```diff
@@ -100,4 +103,29 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(hits, misses, evictions, sizeInBytes, entries);
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        // We don't write the header in CacheStatsResponse's toXContent, because it doesn't know the name of aggregation it's part of
+        builder.humanReadableField(Fields.MEMORY_SIZE_IN_BYTES, Fields.MEMORY_SIZE, new ByteSizeValue(sizeInBytes));
+        builder.field(Fields.EVICTIONS, evictions);
+        builder.field(Fields.HIT_COUNT, hits);
+        builder.field(Fields.MISS_COUNT, misses);
+        builder.field(Fields.ENTRIES, entries);
+        return builder;
+    }
+
+    /**
+     * Field names used to write the values in this object to XContent.
+     */
+    public static final class Fields {
+        public static final String MEMORY_SIZE = "size";
+        public static final String MEMORY_SIZE_IN_BYTES = "size_in_bytes";
+        // TODO: This might not be memory as it could be partially on disk, so I've changed it, but should it be consistent with the earlier
+        // field?
+        public static final String EVICTIONS = "evictions";
+        public static final String HIT_COUNT = "hit_count";
+        public static final String MISS_COUNT = "miss_count";
+        public static final String ENTRIES = "entries";
```

Comment:
This is current number of entries. Could "cache_size" cause confusion with "size" and "size_in_bytes" though? I think the human readable field makes the size in bytes field return something like "size : 40 MB" rather than "size_in_bytes : 40000000" if humanReadable is set.

Reply:
I am not completely sure about the right nomenclature here. @msfroh @sgup432 - What do you guys think about this?

Reply:
Yeah `cache_size` doesn't look right to me though queryCache follows that pattern.
I am inclined towards using something on the lines of ```entries``` though this represents current number of entries.

Reply:
I like `item_count`.